pip/pip-321.md

```diff
@@ -0,0 +1,251 @@
+# PIP-321: Split the responsibilities of namespace replication-clusters
+
+# Background knowledge
+
+Pulsar's geo-replication mechanism is typically used for disaster recovery, enabling the replication of persistently stored message data across multiple data centers. For instance, your application publishes data in one region, and you would like to process it for consumption in other regions. With Pulsar's geo-replication mechanism, messages can be produced and consumed in different geo-replication. See the introduction of geo-replication to get more information.[1]
+A client can set replication clusters for a namespace, and the pulsar broker internally manages replication to all the clusters.
```

Comment:
It's better to also add 

A client can set allowed clusters for a tenant. 

And for this line

We can change to "A client can set replication clusters for a namespace, and the pulsar broker internally manages replication to all the clusters. And the replication clusters for a namespace must be a subgroup of the tenant's allowed clusters."